### ISSUE TITLE
Split tracing/histogram.py into many files

### ISSUE THREAD

**Issue by eakuefner** [catapult-project/catapult#3613]
Currently, tracing/histogram.py is a big monolith, and we want to split it up for ergonomics. This is blocked on  #3612 for reasons described in that bug.

**Comment by benshayden** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=918212
